XenModel/Actions/NRPE/NRPEUpdateAction.cs

```diff
@@ -104,6 +104,28 @@ private void SetNRPEConfigureForHost(IXenObject o)
 
         private void SetNRPEStatus(IXenObject host, bool enableNRPE)
         {
+            // Workaround for CA-384579 (caused by CA-384572):
+            // If the certificate used by NRPE was generated in Yangtze it has no permissions. 
+            // We prevent command (start, restart, etc) failures by calling control, which modifies certificate permissions.
+            // After CA-384572 is resolved we will remove this workaround code.

```

Comment:
This would probably need a version check instead of removal, so it works also for those that delay installing the updates with the fix to said ticket.